handler/src/test/java/io/netty/handler/ssl/SniClientTest.java

```diff
@@ -58,6 +61,60 @@ public void testSniClientOpenSslServerJdkSsl() throws Exception {
         testSniClient(SslProvider.OPENSSL, SslProvider.JDK);
     }
 
+    @Test(timeout = 30000)
```

Comment:
nit is `3000` a constant we use throughout our tests? Can we define this somewhere common?